qlib/DebugProgramControl.qm

```diff
@@ -928,7 +1011,19 @@ public namespace DebugProgramControl {
                 ctx.intrInfo.file = location.file;
                 ctx.intrInfo.line = location.start_line;
                 ctx.intrInfo.endline = location.end_line;
-                #ctx.intrInfo.statement = location;
+                if (location.source != "") {
+                    ctx.intrInfo.source = location.source;
+                }
+                ctx.intrInfo.offset = location.offset;
+                try {
+                    ctx.intrInfo.linecode = getSourceCodeFromFileOrLabel(pgm, (
```

Comment:
Using `{ }` instead of `( )` for non-empty hashed is legacy style from older Qore. Using new style since this commit brings inconsistency. Changing all occurrences in existing code manually is practically non feasible.

Reply:
it was agreed we are going to use the new style from now on (few months ago), and we will update the old code only if necessary for the next development. see https://qoretechnologies.com/devs/tiki-index.php?page=Coding+style 

> Updating existing code
Only change existing code if it makes sense (it is not intended to spent days formatting old code base only to add few new lines).
Update style of existing files lazily, i.e. if you happen to need to modify (adding functionality, fixing bugs, etc.) an existing file and you find its style to be not in accordance with the current style, please fix the style first as a *separate commit* and only then make the functional changes you originally planned.